source/_components/vallox.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: page
+title: "Vallox Ventilation Units"
+description: "Instructions on how to integrate Vallox ventilation units into Home Assistant."
+date: 2019-06-19 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Fan
+  - Sensor
+ha_release: 0.96
+ha_iot_class: Local Polling
+---
```

Comment:
Is there no logo?

Reply:
What are legal implications of adding logos? I have no affiliation to the company whatsoever.

Reply:
We have to check what the company policy on using logos is. Bigger companies usually have instructions on logo use.

Reply:
I'll have to pass on this one, already overshot my budget for working on this integration by too much, sorry :(

Reply:
I have not been able to find any policy, I think you can just add their logo.

Reply:
Will add the logo later.